zeq/detail/port.h

```diff
@@ -19,7 +20,7 @@ namespace detail
 uint16_t getPort( const std::string& name );
 
 /** @return a random port number. */
-uint16_t getRandomPort();
+ZEQ_API uint16_t getRandomPort();
```

Comment:
I agree with that. It should me moved to a common header for tests, unless used in the implementation, which could be the case.